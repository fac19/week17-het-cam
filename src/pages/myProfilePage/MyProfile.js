import React from "react";
import Heading from "../../components/global/heading/Heading";
import Subheading from "../../components/global/subheading/Subheading";
import Navbar from "../../components/global/navbar/Navbar";

const MyProfile = () => {
  return (
    <>
      <Navbar />
      <Heading>My Profile</Heading>
      <Subheading>Coming soon...</Subheading>
    </>
  );
};

export default MyProfile;
