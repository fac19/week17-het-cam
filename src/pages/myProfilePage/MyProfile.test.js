import React from "react";
import { render, screen, BrowserRouter } from "@testing-library/react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import MyProfile from "./MyProfile";
import { BrowserRouter as Router } from "react-router-dom";

test("Landing page component is styled", () => {
  const tree = renderer
    .create(
      <Router>
        <MyProfile />
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test("Profile text renders correctly", () => {
  render(
    <Router>
      <MyProfile />
    </Router>,
  );
  screen.getByText("My Profile");
  screen.getByText("Coming soon...");
});
