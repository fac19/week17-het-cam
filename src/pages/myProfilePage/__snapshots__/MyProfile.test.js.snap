// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing page component is styled 1`] = `
Array [
  .c0 {
  position: fixed;
  top: 0;
  left: 0;
  width: 100vw;
  background-color: #00C2F0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  z-index: 1000;
}

@media (min-width:600px) {

}

@media (min-width:600px) {

}

<nav
    className="c0"
  >
    <a
      aria-label="My missions"
      href="/my-missions"
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root makeStyles-root-1"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
        />
      </svg>
    </a>
    <a
      aria-label="My profile"
      href="/my-profile"
      id="nav-icon-myProfile"
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root makeStyles-root-1"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
        />
      </svg>
    </a>
    <a
      aria-label="Logout"
      href="/"
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root makeStyles-root-1"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M10.09 15.59L11.5 17l5-5-5-5-1.41 1.41L12.67 11H3v2h9.67l-2.58 2.59zM19 3H5c-1.11 0-2 .9-2 2v4h2V5h14v14H5v-4H3v4c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
        />
      </svg>
    </a>
  </nav>,
  .c0 {
  font-family: "Josefin Sans",sans-serif;
  font-size: 2rem;
  color: #032049;
  text-align: center;
}

@media (min-width:600px) {
  .c0 {
    font-size: 4rem;
  }
}

@media (min-width:600px) {

}

<h1
    className="c0"
  >
    My Profile
  </h1>,
  .c0 {
  font-family: "Josefin Sans",sans-serif;
  font-size: 1.5rem;
  color: #032049;
  text-align: center;
  text-transform: uppercase;
}

@media (min-width:600px) {

}

@media (min-width:600px) {
  .c0 {
    font-size: 2.2rem;
  }
}

<h2
    className="c0"
  >
    Coming soon...
  </h2>,
]
`;
